core/services/keeper/validate_test.go

```diff
@@ -54,7 +54,7 @@ func TestValidatedKeeperSpec(t *testing.T) {
 			args: args{
 				tomlString: `
 type            			= "keeper"
-schemaVersion   			= 3
+schemaVersion   			= 1
```

Comment:
do we enforce the schemaVersion in validation?

Reply:
No, it's not needed, so we can remove it 😊